mantidproject/mantid
# Indirect - MSDFit Plot

The buttons for plotting and saving have been disabled until the algorithm is complete and plotting of the results workspace has been updated.
Also the release notes have been updated to show all of the changes to the plotting and saving of Indirect Interfaces

**To test:**

Run MSDFit with \olympic\babylon5\Public\Louise McCann\DataAnalysis\MSDFit
Check plot and save buttons work as expected.

Fixes #17450.



---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
